integration-tests/pom.xml

```diff
@@ -38,6 +38,8 @@
         <sshd.version>2.3.0</sshd.version>
         <testcontainers.version>1.12.5</testcontainers.version>
         <mock-javamail.version>1.9</mock-javamail.version>
+
+        <quarkus.banner.enabled>false</quarkus.banner.enabled>
```

Comment:
Saving a couple of IO bytes?

Reply:
for two reasons:
- it looks ugly at least in my terminal
- some CI may fail if logs are too big

Reply:
> it looks ugly

We should override it with a good looking ASCII Camel!